lib/tier-node.js

```diff
@@ -119,6 +119,26 @@ Reply.prototype.send = function (payload) {
     throw new Error('Reply already sent')
   }
 
+  // If user enabled cors globally and did not disable it on this route
+  if (this.handle.schema && (this.handle.schema.cors || (this.handle.schema.out && this.handle.schema.out.cors))) {
+    // if cors not disabled for this route
+    if (this.handle.schema.out && this.handle.schema.out.cors !== false) {
+      const cors = this.handle.schema.cors || this.handle.schema.out.cors
+
+      const headers = {
+        'Access-Control-Allow-Origin': cors.origin || '*',
+        'Access-Control-Allow-Methods': cors.methods || 'POST,GET,PUT,DELETE,OPTIONS,XMODIFY',
+        'Access-Control-Allow-Credentials': cors.credentials || 'true',
+        'Access-Control-Max-Age': cors.age || '86400',
+        'Access-Control-Allow-Headers': cors.headers || 'X-Requested-With,X-HTTP-Method-Override,Content-Type,Accept'
+      }
+
+      for (let header in headers) {
+        this.res.setHeader(header, headers[header])
+      }
```

Comment:
this is _very_ slow. I would really prefer to avoid the for loop, and rather call `setHeader` for all of them.